src/pages/components/code-block.mdx

```diff
@@ -62,6 +62,17 @@ import LanguageCode from '!!raw-loader!../../examples/typography/code-block/Lang
   <Language />
 </CodeExample>
 
+### Line highlights
+
+Use the `highlightLines` prop to call attention to particular lines of code.
```

Comment:
@ctancinco @steue these are the lines that either need Content Design attention 👍 or 👎

Reply:
i'm no wordsmith, but it looks & makes sense to me!